File: benchmark/comparison/tasks/18/solution.ts
function formatList(arr: number[]): string {
  return "[" + arr.join(", ") + "]";
}

function list_flatten(lists: number[][]): number[] {
  const result: number[] = [];
  for (const list of lists) {
    for (const item of list) {
      result.push(item);
    }
  }
  return result;
}

console.log(formatList(list_flatten([[1, 2], [3], [4, 5]])));
console.log(formatList(list_flatten([[1]])));
console.log(formatList(list_flatten([])));

File: benchmark/comparison/tasks/18/solution.py
def fmt_list(lst):
    return "[" + ", ".join(str(x) for x in lst) + "]"

def list_flatten(lst):
    result = []
    for sub in lst:
        result.extend(sub)
    return result

print(fmt_list(list_flatten([[1, 2], [3], [4, 5]])))
print(fmt_list(list_flatten([[1]])))
print(fmt_list(list_flatten([])))
